tests/dbus/test_dbus_client.cpp

```diff
@@ -338,7 +338,6 @@ int main()
     TEST_ASSERT(region == "US");
 
     TEST_ASSERT(api->GetPreferredChannelMask(preferredChannelMask) == ClientError::ERROR_NONE);
-    TEST_ASSERT(preferredChannelMask == 0x7fff800);
```

Comment:
Could we change it to 

```
TEST_ASSERT(preferredChannelMask == 0x7ff7000);
```

Here is the configuration of the prefered channel mask: https://github.com/zhanglongxia/openthread/blob/master/src/posix/platform/openthread.conf.example#L35

Reply:
The problem with this is that then I guess this PR itself would not pass CI tests (as the OT commit changing it on OT side is being merged from #2025).

Reply:
Agree, we should merge this CL first to make the OpenThread be able to be synchronized to ot-br-posix in https://github.com/openthread/ot-br-posix/pull/2025, then we can submit the changes to check the value of `preferredChannelMask`.